components/edit-collective/sections/SetupTwoFactorAuth.js

```diff
@@ -307,11 +432,27 @@ const twoFactorAuthToAccountMutation = gqlV2/* GraphQL */ `
   }
 `;
 
+const twoFactorAuthFromAccountMutation = gqlV2/* GraphQL */ `
+  mutation RemoveTwoFactorAuthFromAccount($account: AccountReferenceInput!, $code: String!) {
+    removeTwoFactorAuthTokenFromIndividual(account: $account, code: $code) {
+      id
+      ... on Individual {
+        hasTwoFactorAuth
+      }
+    }
+  }
+`;
+
 const addTwoFactorAuthToAccountMutation = graphql(twoFactorAuthToAccountMutation, {
   name: 'addTwoFactorAuthTokenToIndividual',
   options: { context: API_V2_CONTEXT },
 });
 
+const removeTwoFactorAuthFromAccountMutation = graphql(twoFactorAuthFromAccountMutation, {
+  name: 'removeTwoFactorAuthTokenFromIndividual',
```

Comment:
It's confusing that in one case we "removeTwoFactorAuthFromAccount" and in the other "removeTwoFactorAuthTokenFromIndividual". Simply keep "removeTwoFactorAuthFromAccount" for the name.

Reply:
That would require an API change. I'll change them all to 'individual' to keep it consistent on the frontend.

Reply:
The "name" has nothing to do with the API, it's the name of the function that is added to the component.